app/models/potion.rb

```diff
@@ -0,0 +1,82 @@
+# frozen_string_literal: true
+
+class Potion < ApplicationRecord
+  belongs_to :game
+  belongs_to :canonical_potion, optional: true, class_name: 'Canonical::Potion'
+
+  has_many :potions_alchemical_properties, dependent: :destroy, inverse_of: :potion
+  has_many :alchemical_properties, through: :potions_alchemical_properties
```

Comment:
Do we want to scope this like it's scoped in the canonical model, i.e., such that we can use code like this?

```ruby
potion.alchemical_properties.first.strength
potion.alchemical_properties.last.duration
```

Reply:
So far we don't need this so let's not use default scopes when there's no reason to.